keyboards/keycapsss/plaid_pad/keymaps/default/config.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
This file can be deleted.

Reply:
I would like to keep that file.
The reason is, that people can duplicate the default folder and have template to modify as they like.
What do you think? @fauxpark